go/integTest/shared_commands_test.go

```diff
@@ -2990,18 +2990,18 @@ func (suite *GlideTestSuite) TestBLPop() {
 		assert.Nil(suite.T(), err)
 		assert.Equal(suite.T(), int64(2), res1)
 
-		res2, err := client.BLPop(context.Background(), []string{listKey1, listKey2}, float64(0.5))
+		res2, err := client.BLPop(context.Background(), []string{listKey1, listKey2}, 500*time.Millisecond)
```

Comment:
```suggestion
		res2, err := client.BLPop(context.Background(), []string{listKey1, listKey2}, 500 * time.Millisecond)
```
entirely nit, feel free to ignore, i just find having spaces in math ops easier to read

Reply:
me too tbh